src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/SeriesWrapper.tsx

```diff
@@ -92,10 +94,6 @@ const SubSeriesHeaderText = styled(Box)`
   left: 0;
   top: 50px;
   right: 0;
-  mix-blend-mode: difference;
```

Comment:
Is it intentional to remove the blend mode from the header text? We should do this only for mobile.